### ISSUE TITLE
Don't uppercase "Read More" and other footer text in cards

### ISSUE THREAD

**Issue by tofumatt** [mozilla/addons#10796]
Go to https://addons-dev.allizom.org/en-US/android/addon/lastpass-password-manager/ and check out the all uppercase text in the footer cards:

<img width="451" alt="screenshot 2017-09-20 13 31 05" src="https://user-images.githubusercontent.com/90871/30643741-0d045e6a-9e08-11e7-9d1a-1887f083029a.png">
<img width="810" alt="screenshot 2017-09-20 13 31 03" src="https://user-images.githubusercontent.com/90871/30643740-0d031884-9e08-11e7-967a-31dd4f680a52.png">

There are plenty of issues with uppercasing text, so this should be regular case. We could *maybe* use small-caps but even they have accessibility problems so I'd rather not.
